api/FontFace.json

```diff
@@ -272,14 +288,22 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "10"
             },
             "safari_ios": {
               "version_added": null
             },
-            "webview_android": {
-              "version_added": null
-            }
+            "webview_android": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "37",
+                "version_removed": "45",
+                "partial_implementation": true,
+                "notes": "Starting in Chrome 45, the returned promise resolves with a <code>FontFace</code> object as required by the specification instead of void."
```

Comment:
```suggestion
                "notes": "The returned promise resolves void instead of a <code>FontFace</code> object as required by the specification."
```